src/js/shared/config_file.js

```diff
@@ -131,7 +131,7 @@ const ConfigFile = {
 
         // Parse data object.
         const data = JSON.parse(dataRaw);
-        await ConfigFile.parseJSONFile(data, false);
+        await ConfigFile.parseJSONFile(data, true);
```

Comment:
This is wrong: when `updateNow` is called we **don't** want to override all settings, on this case we should use the merging strategies selected by the user.

> `overrideAll=true` is only intended to be used when the user imports a JSON file, or manually edits the JSON file.